clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/api/KubernetesApiAdaptor.groovy

```diff
@@ -41,18 +46,24 @@ class KubernetesApiAdaptor {
   final KubernetesClient client
   final Registry spectatorRegistry
   final Clock spectatorClock
+  final ApiClient apiClient
+  final AppsV1beta1Api apiInstance
 
   public spectatorRegistry() { return spectatorRegistry }
 
-  KubernetesApiAdaptor(String account, io.fabric8.kubernetes.client.Config config, Registry spectatorRegistry) {
-    if (!config) {
+  KubernetesApiAdaptor(String account, io.fabric8.kubernetes.client.Config config, KubernetesApiClientConfig clientConfig, Registry spectatorRegistry) {
+    if (!config || !clientConfig) {
       throw new IllegalArgumentException("Config may not be null.")
     }
     this.config = config
     this.account = account
     this.client = new DefaultKubernetesClient(this.config)
     this.spectatorRegistry = spectatorRegistry
     this.spectatorClock = spectatorRegistry.clock()
+
+    apiClient = clientConfig.getApiCient()
```

Comment:
I think we're going to have an easier time by not conflating both api clients in the same adapter, but to instead use 2 separate adapters.